pre-commit

```diff
@@ -10,6 +10,8 @@ if [ -L "$0" ]; then
 	else
 		DEV_LIB_PATH=$( dirname $( readlink "$0" ) )
 	fi
+elif [ -e "$(dirname "$0")/.dev-lib-path" ]; then
+    DEV_LIB_PATH="$(realpath $(cat "$(dirname "$0")/.dev-lib-path"))"
 else
```

Comment:
`realpath` is *not* available _out of the box_ on Windows, it's not available via `cmd.exe` on my path. I do have it available on my path in PowerShell though courtesy of PortableGit.